### PR TITLE
Adapt article & group sorted order PDF (solves #49)

### PR DESCRIPTION
**Pull request by twothreenine**
Since there weren't any "ready for PR" issues left which I could work on (realistically), I worked on foodcoops/foodsoft#1099.

I also adapted the PDF sorted by groups to show the ordered amount in _group order unit_ and the received amount in _billing unit_.
Matrix PDF remains to be adapted.

If you think this isn't ready to be merged, we can also keep it as a draft and later merge it on upstream (in order to focus on the PR).

### CODE REVIEW

**Comment by twothreenine** [comment]
I don't think this would make the fork "bigger" as it only affects files already affected by the fork.

**Comment by lentschi** [comment]
> I don't think this would make the fork "bigger" as it only affects files already affected by the fork.

Yeah, but it affects more considerably more lines in those files. Also none of this is unit tested and thus neither is you new "total" logic (which btw. has nothing to do with the units fork). 

As for the change itself: It looks good to me in general. I'm just not sure, if everyone would agree with the visible changes in the pdf.

Cons I see (admittedly minor):

1. More text to read - a lot of it repetitive (E.g. if many groups order beer, you will have just as many occurrences of `<amount> Bottle (0.5l)`, which is a bit annoying)
2. Ordered and Received columns are no longer easily comparable. (E.g. for the hackathon "Smoked tofu": `2 Piece (160 g)` |`320.000 g`. For many numbers, this might be arduous.)

Long story short: I'm not saying that in total this PR wouldn't be an improvement, I just think it should rather be done and discussed upstream.
